internal/services/services.go

```diff
@@ -99,7 +99,17 @@ func (svc *DownloadService) Products(params *omnitruck.RequestParams) (data omni
 	request = svc.Omnitruck().Products(params, &data)
 
 	data = svc.DynamoServices(svc.databaseService).Products(data, params.Eol)
-
+	// This a temporary fix to hide CHEF_INFRA_CLIENT_ENTERPRISE_PRODUCT when infra19Enabled is false
+	if !svc.config.SupportInfra19 {
+		// Remove CHEF_INFRA_CLIENT_ENTERPRISE_PRODUCT from the data array
+		var filtered omnitruck.ItemList
+		for _, item := range data {
+			if item != constants.CHEF_INFRA_CLIENT_ENTERPRISE_PRODUCT && item != constants.MIGRATE_ICE {
+				filtered = append(filtered, item)
+			}
+		}
+		data = filtered
+	}
```

Comment:
@rishabhjhs  just create a separate function for doing such kind of post processing after fetching the master list. That way the code would be clean.

Reply:
I don't think function creation is needed here as this is temp things needed for now. Plus functions are generally created where you can reuse it. Which won't happen